repository-setup.md

```diff
@@ -0,0 +1,9 @@
+# Repository Layout
+
+- Master is always Protected.
+- Review where possible must have a second pair of eyes.
+- Squash merging is up to the repository primary maintainer
```

Comment:
Merge strategy (squash vs. merge vs. rebase) is up to the repository's primary maintainer